docs/deployment-arbitrary-cluster.md

```diff
@@ -36,25 +36,8 @@ systemctl start NetworkManager
 
 ## kubernetes-nmstate
 
-Finally, we can install kubernetes-nmstate on our cluster.
-
-### Kubernetes
-
-```shell
-# Install k8s resources and kubernets-nmstate operator
-for manifest in namespace.yaml service_account.yaml role.yaml role_binding.yaml nmstate_v1alpha1_nodenetworkstate_crd.yaml nmstate_v1alpha1_nodenetworkconfigurationpolicy_crd.yaml operator.yaml; do
-    kubectl apply -f https://github.com/nmstate/kubernetes-nmstate/releases/latest/download/$manifest
-done
-```
-
-### OpenShift
-
-```shell
-# Install k8s resources , openshift scc and kubernets-nmstate operator
-for manifest in namespace.yaml service_account.yaml role.yaml role_binding.yaml scc.yaml nmstate_v1alpha1_nodenetworkstate_crd.yaml nmstate_v1alpha1_nodenetworkconfigurationpolicy_crd.yaml operator.yaml; do
-    oc apply -f https://github.com/nmstate/kubernetes-nmstate/releases/latest/download/$manifest
-done
-```
+Finally, we can install kubernetes-nmstate on our cluster. In order to do that,
```

Comment:
This is too much let's just leave this to user, and put two options to install directly from repo (good to test HEAD) and from addons operator (good of stability)

Reply:
We have it, this is for arbitrary cluster. For local cluster they install it from local sources https://github.com/nmstate/kubernetes-nmstate/blob/master/docs/deployment-local-cluster.md